handler.py

```diff
@@ -2,46 +2,45 @@
 from calculations.calculations import Calculations
 from plots.make_plot import make_plot
 import traceback
+import pandas as pd
 
-conversions = Conversions("data\\output2_linpot_2023-10-14_13-23-28.csv")
+conversions = Conversions("data\\output2_linpot_2023-10-14_13-23-28.csv", "data\\output2_analog_2023-10-14_13-18-15.csv")
 calculations = Calculations("data\\output2_linpot_2023-10-14_13-23-28.csv")
+#conversions.clean_data(conversions.data)
 
 
 class handler:
 
     def wheel_load_handler(self):
         # loads and prepares initial data
-        conversions.convert_voltage_to_in()
-        conversions.convert_time(conversions.data)
-        conversions.clean_data()
-        calculations.data = conversions.data
+        # todo: return dataframe
+        df = conversions.data
+        df_xl = conversions.data2
+        df_converted = conversions.convert_voltage_to_in(df)
+        conversions.convert_xl_g()
+        df_cleaned = conversions.clean_data(df_converted)
```

Comment:
can we call the cleaning method prior to this? and clean the voltages before converting ?

Reply:
Let's discuss this. Calling the data cleaning on the voltage increases overall wheel loads by around 2x which does not seem reasonable.